### DISCUSSION TITLE
Tray icon?

### DISCUSSION THREAD

**Discussion by rawyak** [wwmm/easyeffects#2619] [Ideas]
Hello everybody.
I am not a programmer but I use very often PulseEffects, and I just love it.
I was thinking about the fact that having it minimized in a tray icon would optimize the windows managing, while still letting it at hand when needed. 
What do you think about it? Would it be a complicate function to implement?

**Comment by wwmm** [comment]
A tray icon was never implemented in EasyEffects / PulseEffects because in gtk4 removed support for tray icons.

>Would it be a complicate function to implement?

As the graphical toolkit removed them from the workflow it will definitely complicate things.
